react/src/component/PerfTestDashboard.jsx

```diff
@@ -220,17 +230,17 @@ export default class PerfTestDashboard extends React.Component {
 
 
         return <div id='perf_dashboard'>
-            {androidBatteryInfo && <div>
+            {!isAndroidBatteryInfoEmpty && <div>
                 <h3> Android Battery report</h3>
                 {androidBatteryMultiSelect}
                 {renderAndroidBatteryChart}
             </div>}
-            {androidMemoryInfo && <div>
+            {!isAndroidMemoryInfoEmpty && <div>
                 <h3> Android Memory report</h3>
                 {androidMemoryMultiSelect}
                 {renderAndroidMemoryChart}
             </div>}
-            {windowsMemoryInfo && <div>
+            {!isWindowsMemoryInfoEmpty && <div>
```

Comment:
If the `InfoEmpty` is `true`, can we replace the chart with a placeholder and indicate that `There is something wrong when parse the memory data, please check the request param or agent`? We can also ask our team to optimize the placeholder :)

Reply:
Thanks, updated.